### DISCUSSION TITLE
币价高涨，filecoin生态建设，路在何方？

### DISCUSSION THREAD

**Discussion by AllenLuo5292** [filecoin-project/community-china#52] [应用开发]
币价飞涨，质押手续费居高不下，还有人愿意用fil币来质押吗，未来有人愿意用fil做存储，还是更加愿意长期持有，等待未来价值。filecoin生态如何落地，让人担忧？

**Comment by AllenLuo5292** [comment]
蜀道难，难于上青天

**Comment by JeffeyZhou** [comment]
我相信更多的人宁愿要快钱

**Reply by AllenLuo5292** [reply]
是的，赚快钱让这个项目的步履蹒跚~~~

**Comment by h3z** [comment]
挖矿投入产出是按投入多少币，产出多少币。和币价无关。
然后币价是另一回事，其实不会改变挖矿的逻辑。

币价1元，你封装了很多，一天产出1000个币，收益一千块钱。
币价1000元，你封装了很少，一天产1个币，收益也是一千块钱。

币价高低只是在影响你对未来涨跌的判断，这是炒币，跟挖矿没关系。

**Reply by AllenLuo5292** [reply]
666
